### PR TITLE
Add test when no machine manager is set

### PR DESCRIPTION
**Pull request by mkmkl93**
As requested in #24

### CODE REVIEW

File: load_balancer.py [hunk 1]
```diff
@@ -27,6 +30,11 @@ def health_check() -> str:
 def validate_file(request: LinterRequest) -> LinterResponse:
     global linter_number
 
+    print("machine_manager_url", machine_manager_url)
```
Comment (Kerram) [inline load_balancer.py:33]: Remove print

File: load_balancer.py [hunk 2]
```diff
@@ -15,6 +15,9 @@
 lock = Lock()
 
 linter_number = 0
+if "LOAD_BALANCER_MACHINE_MANAGER_URL" in os.environ:
+    logger.exception("LOAD_BALANCER_MACHINE_MANAGER_URL doesn't exist in environment")
```
Comment (Kerram) [inline load_balancer.py:19]: Maybe change this to be different than below. For example by adding "at startup".

File: test/test_load_balancer.py [hunk 3]
```diff
@@ -40,6 +42,15 @@ def test_load_balancer_equal_split_across_one_language(self):
             self.assertEqual(0, len(response.errors))
             self.assertEqual(f"Current responses_count: {i // 2 + 1}", response.test_logging[0])
 
+    def test_when_no_machine_manager_url(self):
+        deploy_utils.stop_fast_api_app(self.load_balancer_process)
+        os.environ.pop("LOAD_BALANCER_MACHINE_MANAGER_URL")
+        self.load_balancer_process, self.load_balancer_url = deploy_utils.start_fast_api_app("load_balancer")
+
+        with self.assertRaises(Exception):
```
Comment (Kerram) [inline test/test_load_balancer.py:50]: Maybe add a check for the exception message?

File: load_balancer.py (hunk 2 above)
Comment (mkmkl93) [reply load_balancer.py:19]: But I am checking if exactly this variable exists

File: test/test_load_balancer.py (hunk 3 above)
Comment (mkmkl93) [reply test/test_load_balancer.py]: Sounds like a good idea

File: load_balancer.py (hunk 1 above)
Comment (mkmkl93) [reply load_balancer.py]: Right, sorry

File: load_balancer.py (hunk 2 above)
Comment (Kerram) [reply load_balancer.py]: Sorry, I meant different than log below (in validate function).

File: test/test_load_balancer.py [hunk 4]
```diff
@@ -47,10 +47,15 @@ def test_when_no_machine_manager_url(self):
         os.environ.pop("LOAD_BALANCER_MACHINE_MANAGER_URL")
         self.load_balancer_process, self.load_balancer_url = deploy_utils.start_fast_api_app("load_balancer")
 
-        with self.assertRaises(Exception):
+        with self.assertRaises(Exception) as cm:
             load_balancer_api.validate(self.load_balancer_url,
                                        E2eTests.flawless_python_request)
 
+        self.assertEqual(
+            "Error in update_manager: {'detail': \"LOAD_BALANCER_MACHINE_MANAGER_URL doesn't exist in environment\"}",
```
Comment (Kerram) [inline test/test_load_balancer.py:55]: You could write self.assertIn("LOAD_BALANCER_MACHINE_MANAGER_URL ... in environment", str(cm.exception)), but this is OK as well.
Thanks!

**Pull request merged**